wren-ui/src/apollo/client/graphql/home.ts

```diff
@@ -152,6 +168,14 @@ export const CREATE_THREAD_RESPONSE = gql`
   ${COMMON_ERROR}
 `;
 
+export const CREATE_THREAD_RESPONSE_EXPLAIN = gql`
```

Comment:
Support error handler for the CreateThreadResponseExplain mutation.

Reply:
Done by [a119eba](https://github.com/Canner/WrenAI/pull/539/commits/a119ebaa339f850821af407eada663e6db1c02f8)